src/signals/incident-management/containers/AreaContainer/AreaContainer.tsx

```diff
@@ -32,13 +33,21 @@ export const AreaContainer: FunctionComponent = () => {
 
   const { data: area, get: getArea } = useFetch<AreaFeatureCollection>()
   const { data: incident, get: getIncident } = useFetch<Incident>()
+  const { data: selectedIncident, get: getSelectedIncident } =
+    useFetch<Incident>()
 
   useEffect(() => {
     getIncident(`${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}`)
     getArea(
       `${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}/context/near/geography`
     )
-  }, [getArea, getIncident, id])
+
+    if (selection) {
+      getSelectedIncident(
+        `${configuration.INCIDENT_PRIVATE_ENDPOINT}${selection.properties.id}`
+      )
+    }
```

Comment:
Better to move this into a separate `useEffect`. Now, every time the selection changes, the two calls above will also be triggered.